.github/workflows/deploy.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           - run: npm ci
 
           - name: update jooby-codec to latest version
-            run: npm install jooby-codec@latest
+            run: npm install jooby-codec@latest @jooby-dev/jooby-codec@latest
```

Comment:
install same packages but different versions, why?

Reply:
discussed

an unused package was deleted

fixed "build" job in workflow "deploy"